Goal App/Constants/Constants.swift

```diff
@@ -10,5 +10,6 @@ import Foundation
 struct Constants {
     enum Storyboard {
```

Comment:
any reason of using an enum vs a separate struct for Storyboard?

Reply:
idea behind this was to separate out the different type of constants I may add in future